utils/src/userInput/showQuickPick.ts

```diff
@@ -236,13 +227,13 @@ function getGroupedPicks<TPick extends types.IAzureQuickPickItem<unknown>>(group
             if (!group.name) {
                 picks.push(...group.picks);
             } else {
+                // TODO: Switch kind from -1 to QuickPickItemKind.Separator when VS Code type files are updated
```

Comment:
We should update the `@types/vscode` package to solve this. Most of our extensions require a minimum VS Code version of `1.62.0` ([example](https://github.com/microsoft/vscode-azurestaticwebapps/blob/main/package.json#L10)). Do you know which version of the types package we need?

We'll want to make sure every extension requires a minimum version of VS Code above the types package we install in the tools package. And we can increase the min version requirement of our extensions if needed.

Reply:
Replying here to make it easier

> I originally planned to do that, but the update to the newest version caused a bunch of other type mismatch issues in un-related files that I thought might make more sense to be captured in a different pull request. Let me know what you think, thanks!

That's odd. Usually not too many things should be changing between versions on the `@types/vscode` package. I'll try it out myself and see what's up.

Reply:
Looks like the feature was added in v1.64.0

Reply:
> Looks like the feature was added in v1.64.0

I think it's worth the effort to do the upgrade to 1.64.0 before merging this PR. We'll also have to bump the minimum versions of VS Code in the extensions to be at least 1.64.0.

Reply:
Agreed. The most current VSCode version is 1.67.2; users generally update quite quickly. By now I'm guessing 99% or more are running 1.64+.